impl/src/metadata.rs

```diff
@@ -91,11 +91,18 @@ impl LockfileGenerator for CargoLockfileGenerator {
     let lockfile_path = crate_root_dir.join("Cargo.lock");
 
     // Generate lockfile
-    std::process::Command::new(&self.cargo_bin_path)
+    let output = std::process::Command::new(&self.cargo_bin_path)
       .arg("generate-lockfile")
       .current_dir(&crate_root_dir)
       .output()?;
```

Comment:
LGTM, though, while you're here and working on error handling, would you mind adding a context to the potential error on this line?

Reply:
Done - thanks!